### PR TITLE
Add missing Jazzy packages

### PR DESCRIPTION
**Pull request by Rayman**
We also need these packages released for Jazzy

### CODE REVIEW

**Comment by Timple** [comment]
I know, but they don't compile yet.

**Comment by Rayman** [comment]
And only the off highway stuff?

File: jazzy.repos [hunk 1]
```diff
@@ -17,6 +17,11 @@ repositories:
     # Until: https://github.com/novatel/novatel_oem7_driver/pull/89
     url: https://github.com/nobleo/novatel_oem7_driver.git
     version: fix/missing-git-dep
+  off_highway_sensor_drivers:
+    type: git
+    # Until: https://github.com/nobleo/off_highway_sensor_drivers
+    url: https://github.com/nobleo/off_highway_sensor_drivers.git
```
Comment (Timple) [inline jazzy.repos:23]: Why our fork? It doesn't contain improvements on the upstream.

Comment (Rayman) [reply jazzy.repos:23]: I just copied from iron.yaml. So you can ask that question to the person who put it there :)

**Comment by Timple** [comment]
So there is an issue with pcl_conversions. Also [spatio_temporal_voxel_layer](https://github.com/SteveMacenski/spatio_temporal_voxel_layer) is affected by this.

We can create a branch on our fork which fixes the dependencies in the meanwhile and build that one?

**Pull request merged**